msg/BatteryState.msg

```diff
@@ -0,0 +1,8 @@
+builtin_interfaces/Time time_stamp # Timestamp from when the message was send
+string sender_id # Name of the node that send this command
+uint32 id # Battery ID, for systems with multiple batteries
```

Comment:
Wouldn't uint8 be sufficient here? Or did you choose this as it is the mavlink type?

Reply:
Yes, because MAVLINK uses a uint32_t for the battery id. See https://mavsdk.mavlink.io/main/en/cpp/api_reference/structmavsdk_1_1_telemetry_1_1_battery.html